sovrin_client/test/cli/conftest.py

```diff
@@ -1055,14 +1055,6 @@ def faberAdded(poolNodesCreated,
 
 
 @pytest.fixture(scope="module")
-def faberIsRunningWithoutNymAdded(emptyLooper, tdirWithPoolTxns, faberWallet,
-                                  faberAgent):
-    faber, faberWallet = runningFaber(emptyLooper, tdirWithPoolTxns,
-                                      faberWallet, faberAgent, None)
-    return faber, faberWallet
-
-
-@pytest.fixture(scope="module")
 def faberIsRunning(emptyLooper, tdirWithPoolTxns, faberWallet,
```

Comment:
Have we decided we no longer need this? How are the same conditions being tested?